packages/edit-site/src/components/resizable-frame/index.js

```diff
@@ -217,28 +258,30 @@ function ResizableFrame( {
 			minWidth={ FRAME_MIN_WIDTH }
 			maxWidth={ isFullWidth ? '100%' : '150%' }
 			maxHeight={ '100%' }
-			onMouseOver={ () => setIsHovering( true ) }
-			onMouseOut={ () => setIsHovering( false ) }
+			onFocus={ () => setShouldShowHandle( true ) }
+			onBlur={ () => setShouldShowHandle( false ) }
+			onMouseOver={ () => setShouldShowHandle( true ) }
+			onMouseOut={ () => setShouldShowHandle( false ) }
 			handleComponent={ {
-				left:
-					isHovering || isResizing ? (
-						<motion.div
+				left: (
+					<Tooltip text={ __( 'Drag to resize' ) }>
+						<motion.button
 							key="handle"
-							className="edit-site-resizable-frame__handle"
+							className={ classnames(
+								'edit-site-resizable-frame__handle',
+								{ 'is-resizing': isResizing }
```

Comment:
Makes `is-resizing` a proper CSS modifier for this element, instead of relying on the modifier on the parent element.